packages/cli/generation/remote-generation/remote-workspace-runner/src/publishDocs.ts

```diff
@@ -218,9 +218,54 @@ async function convertDocsConfiguration({
         },
         colors: docsDefinition.config.colors,
         navbarLinks: docsDefinition.config.navbarLinks,
+        typography: convertDocsTypographyConfiguration({
+            typographyConfiguration: docsDefinition.config.typography,
+            docsDefinition,
+            uploadUrls,
+            context,
+        }),
     };
 }
 
+function convertDocsTypographyConfiguration({
+    typographyConfiguration,
+    docsDefinition,
+    uploadUrls,
+    context,
+}: {
+    typographyConfiguration?: TypographyConfig;
+    docsDefinition: DocsDefinition;
+    uploadUrls: Record<FernRegistry.docs.v1.write.FilePath, FernRegistry.docs.v1.write.FileS3UploadUrl>;
+    context: TaskContext;
+}): FernRegistry.docs.v1.write.DocsTypographyConfig | undefined {
+    if (typographyConfiguration == null) {
+        return;
+    }
+    const result: FernRegistry.docs.v1.write.DocsTypographyConfig = {};
+
```

Comment:
This makes me a little nervous, eg if we add another key to the typography config or there happens to be an extra key at runtime. Can we explicitly convert the three fonts?

Reply:
👍🏽

Reply:
thank you!!